sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/aio/_form_training_client_async.py

```diff
@@ -85,19 +85,19 @@ def __init__(
         authentication_policy = get_authentication_policy(credential)
         self._client = FormRecognizer(
             endpoint=self._endpoint,
-            credential=self._credential,
+            credential=self._credential,  # type: ignore
             sdk_moniker=USER_AGENT,
             authentication_policy=authentication_policy,
             **kwargs
         )
 
     @distributed_trace_async
-    async def train_model(
+    async def begin_training(
             self,
             training_files_url: str,
             use_training_labels: bool,
             **kwargs: Any
-    ) -> CustomFormModel:
+    ) -> AsyncLROPoller:
```

Comment:
I believe latest azure-core will allow you to do AsyncLROPoller[CustomFormModel] in typing.

Same goes for the sync pollers, but that can be another PR

Reply:
Cool! Let me try